NEWS.rst

```diff
@@ -67,6 +67,10 @@ A parser has been added for parsing PDBML (PDB XML) files.
 `PDBML <https://pdbml.wwpdb.org/>`_ is a representation of PDB data in XML format.
 The PDB chapter of the tutorial is updated to show how to use the PDBML parser.
 
+Additionally, a parser has been added for BinaryCIF files.
+BinaryCIF is a compact, binary representation of CIF data.
```

Comment:
I'd add a blurb saying that this format will replace MMTF as the default binary format offered by the RCSB PDB.